db/migrate/20220225184027_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.integer :authorId
+      t.string :title
+      t.text :text
+      t.integer :commentsCount
+      t.integer :likeCount
```

Comment:
- [ ] Let's also follow the ruby best practice for naming convention here. Reason as given above.

Reply:
Thanks for yorr comments @enaburekhan!